doc/lsst.analysis.tools/plot-types.rst

```diff
@@ -0,0 +1,13 @@
+Plot Types
+==========
+Details of current plot types are here, these plots are all in ``python/lsst/analysis/tools/actions/plot`` and
+can all be found on `github <https://github.com/lsst/analysis_tools/blob/main/python/lsst/analysis/tools/actions/plot/>`__.
+
+
+.. automodapi:: lsst.analysis.tools.actions.plot
+   :no-inherited-members:
```

Comment:
```suggestion
.. automodapi:: lsst.analysis.tools.actions.plot
   :no-inherited-members:
   :noindex:
```

This this; I'm not certain it'll work but might solve the warnings about duplicate documentation.

Reply:
It doesn't solve the warnings and it breaks things elsewhere. I'm not sure what to do about this, but if it won't break anything I will merge it and fix it later.